Lib/Rabbit4000/BIOSLIB/CPUPARAM.LIB

```diff
@@ -268,21 +268,21 @@ END DESCRIPTION ***************************************************************/
 extern int exp_ok;
 extern int exp_not_ok;
 #endif
-void setEXPCanonical(void);
+root void setEXPCanonical(void);

```

Comment:
Although it isn't already documented in the function, I think we should list the registers that get trashed (`JKHL`, `JKHL'`, `BCDE`, `PW'`), along with the fact that it will preserve `PW`, even if it swaps the `Px` registers.